apps/stress-test/package.json

```diff
@@ -0,0 +1,68 @@
+    "bench:simple-stress": "yarn bench:simple-stress:mount && yarn bench:simple-stress:inject-styles && yarn bench:simple-stress:prop-update && yarn bench:simple-stress:add-node && yarn bench:simple-stress:remove-node",
+    "bench:simple-stress:process": "node ./benchmarks/simple-stress/processResults.js",
+    "bench:simple-stress:mount": "yarn bench:simple-stress:mount:chrome && yarn bench:simple-stress:mount:firefox",
+    "bench:simple-stress:inject-styles": "yarn bench:simple-stress:inject-styles:chrome && yarn bench:simple-stress:inject-styles:firefox",
+    "bench:simple-stress:prop-update": "yarn bench:simple-stress:prop-update:chrome && yarn bench:simple-stress:prop-update:firefox",
+    "bench:simple-stress:add-node": "yarn bench:simple-stress:add-node:chrome && yarn bench:simple-stress:add-node:firefox",
+    "bench:simple-stress:remove-node": "yarn bench:simple-stress:remove-node:chrome && yarn bench:simple-stress:remove-node:firefox",
+    "bench:simple-stress:mount:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.mount.json --json-file ./benchmarks/simple-stress/results/chrome.mount.results.json",
+    "bench:simple-stress:mount:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.mount.json --json-file ./benchmarks/simple-stress/results/firefox.mount.results.json",
+    "bench:simple-stress:inject-styles:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.inject-styles.json --json-file ./benchmarks/simple-stress/results/chrome.inject-styles.results.json",
+    "bench:simple-stress:inject-styles:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.inject-styles.json --json-file ./benchmarks/simple-stress/results/firefox.inject-styles.results.json",
+    "bench:simple-stress:prop-update:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.prop-update.json --json-file ./benchmarks/simple-stress/results/chrome.prop-update.results.json",
+    "bench:simple-stress:prop-update:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.prop-update.json --json-file ./benchmarks/simple-stress/results/firefox.prop-update.results.json",
+    "bench:simple-stress:add-node:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.add-node.json --json-file ./benchmarks/simple-stress/results/chrome.add-node.results.json",
+    "bench:simple-stress:add-node:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.add-node.json --json-file ./benchmarks/simple-stress/results/firefox.add-node.results.json",
+    "bench:simple-stress:remove-node:chrome": "tach --config ./benchmarks/simple-stress/config/chrome.remove-node.json --json-file ./benchmarks/simple-stress/results/chrome.remove-node.results.json",
+    "bench:simple-stress:remove-node:firefox": "tach --config ./benchmarks/simple-stress/config/firefox.remove-node.json --json-file ./benchmarks/simple-stress/results/firefox.remove-node.results.json"
+  },
+  "devDependencies": {
```

Comment:
we should follow single version policy and put dev dependencies in root package.json

Reply:
How does that work? Do I just put all the devDeps into the root package.json?

Follow up/related question: I've found that I need a different version of `ts-loader` than the one used in the root package.json. I plan to look into it more but how do we handle cases like this assuming I need to stick with a different `ts-loader` version?

Reply:
> How does that work? Do I just put all the devDeps into the root package.json?

Yup more or less

> but how do we handle cases like this assuming I need to stick with a different ts-loader version?

Generally we shouldn't use more than one version of a dependency in our repo, I'd be interested to know why you can't downgrade/upgrade the current shared ts-loader version to match yours

Reply:
I spent some time looking into the version issue and was going to recommend that we just go ahead and upgrade `ts-loader` for the entire repo when I saw this :)
<img width="882" alt="ts-loader" src="https://user-images.githubusercontent.com/93940821/184178402-1f640546-29e5-4f77-9bc7-17f75d5c23bb.png">

I still don't know _why_ the 8.x version had issues with the web components lib. As far as I can tell the big change between `ts-loader` 8 and 9 was that[ 8's Webpack 5 support was "best effort" and 9 has full Webpack 5 support (and it dropped Webpack 4 support).](https://blog.johnnyreilly.com/2021/04/20/ts-loader-goes-webpack-5).